### ISSUE TITLE
After some time some of the settings from the admin panel are ignored (links, images, display)

### ISSUE THREAD

**Issue by Softvision-GeluHaiduc** [mozilla/hubs-cloud#175]
**[Affected Versions]:**
- Firefox Release 84.0.2

**[Affected Platforms]:**
- Windows 10x64

**[Prerequisites]:**
- Make sure to be logged in as an admin and have the Admin Panel opened.

**[Steps to reproduce]:**
1. Click the "App Settings" list item from the left side menu in the "Setup" section.
2. Click the "Features" tab on the page.
3. Switch the “Hide Powered By Hubs” setting and save the changes.
4. Go to the cloud instance homepage, scroll down to the bottom and observe the bottom-left corner.

**[Expected results]:**
- The “Powered by Hubs Cloud” message is not displayed.

**[Actual results]:**
- The “Powered by Hubs Cloud” message is still displayed.

**[Notes]:**
- The issue is reproducible with the following settings as well: Show Terms of Use, Show Privacy Notice, Show Controls Links, Show Docs Links, Show Features Links, Show Community Links, Show Company Logo, Show Issue Report Link, Show Avatar Pipelines Links, Show Model Collection Link.
- The issue is also reproducible with the custom images that can be uploaded.
- Restarting the instance does not resolve the problem.
- This issue appeared only after some time using the hubs cloud instance.
- Attached a screen recording of the issue below
![links](https://user-images.githubusercontent.com/51400741/105455924-04773f00-5c8d-11eb-847b-369e230aa988.gif)

**Comment by johnshaughnessy** [comment]
This should block release until we understand what is happening here. (Should I add the `[QA]:Blocker issue` label?)

**Comment by Softvision-GeluHaiduc** [comment]
The reasons I didn’t tag it as a `[QA]:Blocker issue` is because of the following:
- Only I reproduced the issue using my stack
- I was able to reproduce the issue only twice while testing the Admin Panel for a longer period of time (maybe I was "stressing" the admin panel)  
- The issue was no longer reproducible after rebuilding the stack and testing other areas.

Hope this information helps in understanding the issue, I couldn't identify specific steps to reproduce the breakage.
However, I think that we could go with your suggestion and add the `[QA]:Blocker issue` label until further investigations, and the source of the problem is found.

**Comment by jsmoz** [comment]
I noticed that some of the app settings did not always take immediate effect. Instead it seemed to have taken a few refreshes (20-30 seconds) before the new settings were applied (e.g. toggling "Hide Powered By Hubs" or "Show Community Links", etc). 

However, I did not reproduce the issue in a way that it got "stuck" with the wrong settings. Let's remember this issue and revive this discussion if it happens again. For now it will not be a blocker.

**Comment by johnshaughnessy** [comment]
I noticed that some of the app settings did not always take immediate effect. Instead it seemed to have taken a few refreshes (20-30 seconds) before the new settings were applied (e.g. toggling "Hide Powered By Hubs" or "Show Community Links", etc).

However, I did not reproduce the issue in a way that it got "stuck" with the wrong settings. Let's remember this issue and revive this discussion if it happens again. For now it will not be a blocker.

**Comment by Softvision-GeluHaiduc** [comment]
Yes sometimes I saw this as well but I also let the changed settings over night to see if it updates but still nothing. 
I only saw the 30 seconds delay at the EOD for us.

**Comment by mattpolanieckidev** [comment]
Hello,

I am seeing the images are not properly loading on the homescreen. Likely to the encoding being incorrect. Do you have any suggestions?

![image](https://user-images.githubusercontent.com/10778873/139494551-dc038927-52a4-4f30-aa13-d48aa9c10ef3.png)
